paxml/trainer_lib.py

```diff
@@ -1130,8 +1130,15 @@ def train_step_single_learner(
     wps_with_opt = tasks_lib.filter_vars_for_grad_or_opt(
         var_weight_hparams, excluded_for_opt
     )
```

Comment:
should this be in this PR?
(maybe diff was set incorrectly for a sequential PRs)?

Reply:
No, this PR is based on [the first PR](https://github.com/google/paxml/pull/48). This PR is essentially only this file: paxml/tasks/lm/model_params.py. Anyway, I think I will remove this PR and merge it to the first PR.